core/extensions.js

```diff
@@ -264,8 +253,19 @@ Blockly.Extensions.checkBlockHasMutatorProperties_ = function(errorPrefix,
  */
 Blockly.Extensions.getMutatorProperties_ = function(block) {
```

Comment:
Null used to be an option, but not after this change.  Therefore the return signature is now {!Array.<!Object>} or {!Array.<!Function>}